src/LiveSplit.Timer/UI/Components/TimerSettings.Designer.cs

```diff
@@ -104,10 +105,12 @@ private void InitializeComponent()
             this.cmbAccuracy.Items.AddRange(new object[] {
             "",
             ".2",
-            ".23"});
-            this.cmbAccuracy.Location = new System.Drawing.Point(313, 232);
+            ".23",
+            ".234"});
```

Comment:
This seems like the only change you need in this PR. I'm not sure why you have so many unrelated changes in here

Reply:
I didn't change that, it was the Designer that did it